src/main/webapp/index.html

```diff
@@ -25,7 +25,7 @@
   <body onload="addLoginOrLogoutLinkToNavigation();">
     <nav>
       <ul id="navigation">
-        <li><a href="/">Home</a></li>
```

Comment:
I would maaaybe keep the link to home on this page as well, just so the navigation is somewhat consistent? I don't feel super strongly though.